sct-commons/src/main/java/org/lfenergy/compas/sct/commons/ExtRefEditorService.java

```diff
@@ -331,6 +331,12 @@ private void updateLDEPFExtRefBinding(TExtRef extRef, TIED iedSource, TChannel s
         }
         String doName = isBlank(setting.getDOInst()) || setting.getDOInst().equals("0") ? setting.getDOName() : setting.getDOName() + setting.getDOInst();
         extRef.setDoName(doName);
+        if(isExternal && setting.getChannelType().equals(TChannelType.DIGITAL)){
+            extRef.setServiceType(TServiceType.GOOSE);
+        }
+        if(isExternal && setting.getChannelType().equals(TChannelType.ANALOG)){
+            extRef.setServiceType(TServiceType.SMV);
+        }
```

Comment:
I suggest to mutualize the test on isExternal and use a switch case : 
```suggestion
        if (isExternal) {
            extRef.setServiceType(
                    switch (setting.getChannelType()) {
                        case DIGITAL -> TServiceType.GOOSE;
                        case ANALOG -> TServiceType.SMV;
                    });
        }
```